src/main/java/tutorease/address/logic/commands/DeleteContactCommand.java

```diff
@@ -46,16 +51,19 @@ public DeleteContactCommand(Index targetIndex) {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
+        logger.log(Level.INFO, "Executing Delete Contact Command");
         requireNonNull(model);
         List<Person> lastShownList = model.getFilteredPersonList();
 
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            logger.log(Level.WARNING, "Invalid target index: {0}", targetIndex.getOneBased());
```

Comment:
In looginf, should we use oneBased index or zero based?

Reply:
i think maybe zero based as the check if for zero based

Reply:
okay sure